base_partner_merge/base_partner_merge.py

```diff
@@ -371,16 +371,16 @@ def _merge(self, cr, uid, partner_ids, dst_partner=None, context=None):
             src_partners = ordered_partners[:-1]
         _logger.info("dst_partner: %s", dst_partner.id)
 
-        if (openerp.SUPERUSER_ID != uid
-                and self._model_is_installed(cr, uid, 'account.move.line',
-                                             context=context)
-                and self.pool.get('account.move.line'
-                                  ).search(cr, openerp.SUPERUSER_ID,
-                                           [('partner_id',
-                                             'in',
-                                             [partner.id for partner
-                                              in src_partners])],
-                                           context=context)):
+        if (openerp.SUPERUSER_ID != uid and
+                self._model_is_installed(cr, uid, 'account.move.line',
+                                         context=context) and
+                self.pool.get('account.move.line'
```

Comment:
not too keen in this. The goal here is to get a green 8.0 branch, not to fix bad code everywhere.

I'd implement that in a separate PR.